angularSTEP/src/app/navbar/navbar.component.scss

```diff
@@ -1,22 +1,19 @@
-.spacer{
-    flex: 1 1 auto;
-}
-
-a:hover{
-    text-decoration: none;
-    color: red;
-}
-
-a:active{
-    text-decoration: none;
-    color: red;
-}
-
-a:visited{
-    text-decoration: none;
-    color: red;
+.search-input {
+    font: 400 14px/20px Roboto, "Helvetica Neue", sans-serif;
+    font-size: 16px;
+    background: #f5f5f5;
+    color: rgba(0, 0, 0, 0.87);
+    border: 1.5px solid rgba(0, 0, 0, 0.87);
+    outline: none;
+    height: 35px;
+    border-radius: 10px;
+    padding: 5px 12px;
+    margin: 0;
+    width: 200px;
+    vertical-align: bottom;
+    text-align: inherit;
 }
 
-a{
-    border-radius: none;
-}
\ No newline at end of file
+.example-spacer {
```

Comment:
Is "example" real terminology you intend to use, or a placeholder word?

Reply:
Oops... :) for this and for the below comment on flex, I was looking around on the Angular component documentation. For the navbar, to get the menu to the right, they use this flex schema. It seems that `float: right` does not work within the mat-toolbar